src/styles/card.scss

```diff
@@ -0,0 +1,106 @@
+$text-color: #616070;
+$button-text-color: #00acdc;
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  box-sizing: border-box;
+  border: solid 1px #f3f3f3;
+  border-radius: 5px;
+  padding-inline: 16px;
+  transition: all 300ms linear;
+
+  &:hover{
+    transform: scale(1.2);
+  }
```

Comment:
According to the BEM methodology, it's better to avoid nesting elements inside blocks. Instead, you should create a new class for the hover state of the card. For example, you can use `.card--hover` or `.card_hovered`.